stylesheets/github-dark.css

```diff
@@ -1,116 +0,0 @@
-/*
```

Comment:
I see we've reverted back to the default jekyll theme over the one we had before. That's OK for now, let's get the content right, then put some design thought into it.